applovin_max/lib/src/enums.dart

```diff
@@ -71,3 +71,56 @@ enum UserGender {
   /// @nodoc
   const UserGender(this.value);
 }
+
+///
+/// This enum represents the user's geography used to determine the type of
+/// consent flow shown to the user.
+///
+enum ConsentFlowUserGeography {
```

Comment:
We've deprecated the `ConsentDialogState` in favor of this. So, we should do the same here.